app_dart/lib/src/request_handlers/check_for_waiting_pull_requests.dart

```diff
@@ -287,12 +287,12 @@ class CheckForWaitingPullRequests extends ApiRequestHandler<Body> {
     log.info('Validating name: $name, branch: $branch, checks: $checkRuns');
     for (Map<String, dynamic> checkRun in checkRuns) {
       final String? name = checkRun['name'] as String?;
-      if (checkRun['status'] != 'COMPLETED') {
-        allSuccess = false;
-      } else if (checkRun['conclusion'] != 'SUCCESS') {
-        allSuccess = false;
+      if (checkRun['conclusion'] == 'SUCCESS') {
```

Comment:
I was initially looking for this issue in cocoon's luci-status logic, but it's not obvious to me what would cause a check to get into this state. As a path forward, I decided to update this code here to handle it.

Agreed that a running success check needs more investigation, but my understanding is when success is set that is the final state we care about.